app/services/users/omniauth.rb

```diff
@@ -33,5 +31,6 @@ def create_user
     )
     user.skip_confirmation!
     user.save
+    user
```

Comment:
@sinpai isn't `#save` return object? From your code, it seems to me that it returns bool

Reply:
It was cool, so I added user as return object... I will check that